src/Core/Components/MessageBar/Services/IMessageService.cs

```diff
@@ -16,25 +16,43 @@ public interface IMessageService : IDisposable
     Message ShowMessageBar(Action<MessageOptions> options);
 
     /// <summary />
-    Message ShowMessageBar(string message);
+    Message ShowMessageBar(string title);
 
     /// <summary />
-    Message ShowMessageBar(string message, MessageIntent intent);
+    Message ShowMessageBar(string title, bool allowDismiss);
 
     /// <summary />
-    Message ShowMessageBar(string message, MessageIntent intent, string section);
+    Message ShowMessageBar(string title, MessageIntent intent);
```

Comment:
Why rename `message` to `title`?

Reply:
The implementation (`MessageService`) had the parameter named `title` so I was merely updating it for consistency.  If the better name is `message` then I will update everything to be `message.

Reply:
It appears that the underlying `Options` object has a `Title` property on it that the `message` parameter is being mapped to... perhaps that's why the parameter was named `title` on the implementation side.  Either way, I think the interface and the implementation should have the same name.  What would you like that to be?

Reply:
I want to follow the design guidelines as much as possible:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/62f471e6-23dc-4621-9c8d-46f7b1734dc7)

We are already streching that with th option to not have the close button

Going by what we are doing in the example 'Add on top',
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/f30ddf9e-8087-44e7-ae4c-5eb0b2eeaec8)

I would say that that is showing the Title (number 3 from the design image). So I think `MessageService.cs` is correct and `IMessageService.cs` needs to be changed. Agree?

Reply:
Yes. Correct

Reply:
Both the `IMessageService` and the `MessageService` implementation use `title` now.